src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age = undefined, sex, isMarried, partnerName } = person;
+
+  const ageText =
+    age !== undefined ? <p className="Person__age">I am {age}</p> : null;
```

Comment:
don't need to set default value as undefined, and strictly compare with it. Just check if a value exists and actually you don't need to set null. In react you can render conditionally directly in jsx:
 ```jsx
{age && <p className="Person__age">I am {age}</p>}
```